src/sgame/sg_main.cpp

```diff
@@ -524,7 +524,10 @@ void G_InitGame( int levelTime, int randomSeed, bool inClient )
 	G_admin_readconfig( nullptr );
 
 	// initialize all entities for this game
-	memset( g_entities, 0, MAX_GENTITIES * sizeof( g_entities[ 0 ] ) );
+	for ( int i=0; i < MAX_GENTITIES; i++ )
+	{
+		new(&g_entities[i])gentity_t{};
```

Comment:
So the placement new seems to crash nacl-clang. You can try parentheses instead of curly braces there - I think that worked to stop MSVC crashes in the past...

Reply:
src/cgame/cg_main.cpp line 1118 contains `new(&cgs) cgs_t{}; // Using {} instead of () to work around MSVC bug`. Which is why I've used `{}` here

Reply:
Well, that was a long-obsolete MSVC version so we shouldn't have to worry about it anymore...
Anyway gentity_t is only 3K in size, so we don't need to worry about it overflowing the stack. `g_entities[i] = {}` would work